apps/niftest/niftest.cpp

```diff
@@ -18,106 +16,88 @@ namespace bpo = boost::program_options;
-    std::string extension = filename.substr(filename.find_last_of(".")+1);
+bool hasExtension(const std::string &filename, std::string extensionToFind) {
+    std::string extension = filename.substr(filename.find_last_of('.') + 1);
 
     //Convert strings to lower case for comparison
     std::transform(extension.begin(), extension.end(), extension.begin(), ::tolower);
     std::transform(extensionToFind.begin(), extensionToFind.end(), extensionToFind.begin(), ::tolower);
 
-    if(extension == extensionToFind)
-        return true;
-    else
-        return false;
+    return extension == extensionToFind;
 }
 
 ///See if the file has the "nif" extension.
-bool isNIF(const std::string & filename)
-{
-    return hasExtension(filename,"nif");
+bool isNIF(const std::string &filename) {
+    return hasExtension(filename, "nif");
 }
+
 ///See if the file has the "bsa" extension.
-bool isBSA(const std::string & filename)
-{
-    return hasExtension(filename,"bsa");
+bool isBSA(const std::string &filename) {
+    return hasExtension(filename, "bsa");
 }
 
 /// Check all the nif files in a given VFS::Archive
 /// \note Takes ownership!
 /// \note Can not read a bsa file inside of a bsa file.
-void readVFS(VFS::Archive* anArchive,std::string archivePath = "")
-{
+void readVFS(VFS::Archive *anArchive, const std::string &archivePath = "") {
     VFS::Manager myManager(true);
     myManager.addArchive(anArchive);
     myManager.buildIndex();
 
-    std::map<std::string, VFS::File*> files=myManager.getIndex();
-    for(std::map<std::string, VFS::File*>::const_iterator it=files.begin(); it!=files.end(); ++it)
-    {
+    std::map<std::string, VFS::File *> files = myManager.getIndex();
+    for (std::map<std::string, VFS::File *>::const_iterator it = files.begin(); it != files.end(); ++it) {
         std::string name = it->first;
 
-        try{
-            if(isNIF(name))
-            {
-            //           std::cout << "Decoding: " << name << std::endl;
-                Nif::NIFFile temp_nif(myManager.get(name),archivePath+name);
-            }
-            else if(isBSA(name))
-            {
-                if(!archivePath.empty() && !isBSA(archivePath))
-                {
+        try {
+            if (isNIF(name)) {
+                //           std::cout << "Decoding: " << name << std::endl;
+                Nif::NIFFile temp_nif(myManager.get(name), archivePath + name);
+            } else if (isBSA(name)) {
+                if (!archivePath.empty() && !isBSA(archivePath)) {
 //                     std::cout << "Reading BSA File: " << name << std::endl;
-                    readVFS(new VFS::BsaArchive(archivePath+name),archivePath+name+"/");
+                    readVFS(new VFS::BsaArchive(archivePath + name), archivePath + name + "/");
 //                     std::cout << "Done with BSA File: " << name << std::endl;
                 }
             }
         }
-        catch (std::exception& e)
-        {
+        catch (std::exception &e) {
```

Comment:
yeah, pls don't do this

Reply:
Is it Allman or GTFO, or possibility of finally going through with a 'style' brush and cleaning up so that everything is consistent?

I'm partial to K&R, what's the consensus...

Reply:
Allman.